js/packages/deploy/package.json

```diff
@@ -6,9 +6,11 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "Bobby Martin",
+  "author": "Nugget Industries",
   "license": "ISC",
   "dependencies": {
-    "clp": "^4.0.8"
+    "remote-exec": "0.0.3",
+    "scp2": "^0.5.0",
+    "yargs": "^11.0.0"
```

Comment:
So I would add the stuff I find for controlling magnetometers and stuff here?

Reply:
Magnetometer stuff would go somewhere in [here](https://github.com/Wentworth-IEEE/rovsquad18/tree/better-deployment/js/remote)